libr/core/rvc.c

```diff
@@ -535,7 +535,7 @@ R_API int r_vc_git_add (const char *path, const char *fname) {
 		free (cwd);
 		return -2;
 	}
-	ret = r_sys_cmdf ("pwd; git add %s", fname);
+	ret = r_sys_cmdf ("pwd; git add '%s'", fname);
```

Comment:
uhm better to escape, single quotes dont work well on windows.also i think this pwd can go away

Reply:
Ah yeah used it for debugging sorry ;)